File: CS50-main/PS6/sentimental-mario-more/mario.py
# TODO
import cs50

N = cs50.get_int("Height: ")

while N < 1 or N > 8:
    N = cs50.get_int("Height: ")

if N == 1:
    print("#  #")

if N == 2:
    print(" #  #")
    print("##  ##")

File: CS50-main/PS1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Looping question that keeps on asking question until an appropriate answer is received
    int n;
    do
    {
        n = get_int ("Height: ");
    }
    while (n < 1 || n > 8);

    //Building both towers

    for ( int i = 0; i < n; i++)
    {
        for ( int e = 0 ; e < n - (i + 1) ; e++)
        {
            printf(" ");
        }
        for ( int e = 0 ; e < i + 1 ; e++)
        {
            printf("#");
        }
        printf("  ");
        for ( int e = 0 ; e < i + 1 ; e++)
        {
            printf("#");
        }
        printf("\n");
    }

}